extension.js

```diff
@@ -377,19 +413,25 @@ async function executePrimaryTask(context) {
     return;
   }
 
-
   let cases;
-  const fileExtension = filepath
-    .split(".")
-    .pop()
-    .toLowerCase();
-  if (!(fileExtension === "cpp" || fileExtension === "c")) {
+  const fileExtension = getExtension(filePath);
+  const language = getLanguage(fileExtension);
+
+  const validExtensions = Object.values(config.extensions);
+  const extList = validExtensions.map(ext => `.${ext}`);
+  const extListPresentation = `${
+    extList.slice(0, -1).join(', ')
+  } or ${
+    extList.slice(-1)[0]
+  }`;
+
+  if (!validExtensions.includes(fileExtension)) {
     vscode.window.showInformationMessage(
-      "Active file must be have a .c or .cpp extension"
+      `Active file must be have a ${extListPresentation} extension`
```

Comment:
nit: You can add a period `.` at the end.

Reply:
Do you mean at the end of the sentence?